kroxylicious-operator/examples/record-encryption/README.md

```diff
@@ -0,0 +1,12 @@
+These manifests declare a Kafka proxy which does record encryption.
+
+For the purposes of this example we use
+* an in-cluster Kafka provided by [Strimzi](https://strimzi.io/)
+* an in-cluster KMS provided by Hashicorp Vault
+
+As such this example should work more-or-less any Kube cluster which is able to pull the necessary images. But note that Strimzi also supports other KMSes.
+
+To try this example out:
+1. Install kubectl
+2. `cd` to this directory
+3. `kubectl apply -f .`
```

Comment:
looks like these two examples aren't quite complete yet, `record-encryption` doesn't have a filter and `record-validation` is using a RecordEncryption filter.

Reply:
Yes, these are 'aspirational' at this point. We'd need to figure out things like installing Strimzi and a KMS etc. I think we should be aiming for something like this though. I'm inclined to add them anyway, with a `**TODO: Implement this**` and file an issue to actually fill them out. Wdyt?

Reply:
yep a TODO sounds good for the ones that need other bits of plumbing. Though if we want something immediately just to show the Filter wiring working we could set RecordValidation up just checking JSON syntax

Reply:
ah all of them need a kafka though, yeah happy to leave it for later